input/docs/handbook/data-binding/xamarin-forms.md

```diff
@@ -49,6 +52,7 @@ public partial class TheContentPage : ReactiveContentPage<TheViewModel>
     public ThePage()
     {
         InitializeComponent();
+        ViewModel = new TheViewModel();
```

Comment:
So the only issue I have is with this, if the platform activates this view, it will set the view model.  This will explicitly override the previous set view model by the platform.
I personally haven't had to explicitly set the view model unless I am not using the default `RoutingState`.

Reply:
The reason why I'd argue it's better to leave that line here is that in this example the `Locator` class is not mentioned, we are not registering the ViewModel anywhere here. So newbies may wonder why the given code snippets won't work etc. 

If we remove this line, might be worth mentioning how to correctly register the `TViewModel` and `IViewFor<TViewModel>` to make the example "just work". Please feel free to push directly into this branch if you are willing to explain `Locator` usage, might be really worth it. :+1:

Reply:
Reading it I would think that I always have to new up a view model in my view, which isn't the case.  I only need to do this when I am not using the platform as designed.  This will also force a binding context changed event, which may or may not be desired.

I added https://github.com/reactiveui/website/issues/212 and https://github.com/reactiveui/website/issues/213 to track the need to document these platform features.